cg/meta/orders/ticket_handler.py

```diff
@@ -52,7 +52,9 @@ def create_ticket(
         return ticket_nr
 
     def create_new_ticket_message(self, order: OrderIn, user_name: str, project: str) -> str:
-        message = f"data:text/html;charset=utf-8,New incoming {project} samples: "
+        message = (
+            f"data:text/html;charset=utf-8, {len(order.samples)} new incoming {project} samples: "
```

Comment:
Perhaps we remove the "new" since not all orders put are containing new samples, but can be re-runs or addition of parents to index patients

Reply:
fixed